frontend/src/features/mapSideBar/SidebarContents/BuildingForm.test.tsx

```diff
@@ -40,11 +40,17 @@ jest.mock('@react-keycloak/web');
   },
 });
 
-const getBuildingForm = disabled => {
+const getBuildingForm = (disabled: boolean) => {
   return (
     <Provider store={store}>
       <Router history={history}>
-        <BuildingForm setMovingPinNameSpace={noop} nameSpace="building" disabled={disabled} />
+        <BuildingForm
```

Comment:
this was just missing some props and throwing errors

Reply:
Wondering if we can make some of these props for Building Form optional if we are going to be passing `noop`